### DISCUSSION TITLE
[python] How to set multiple (accepted) languages?

### DISCUSSION THREAD

**Discussion by tobwen** [daijro/camoufox#37] [Q&A]
### My question:
How can I set multiple (accepted) lanuages in Camoufox?

### The problem:
That's what a normal, fresh installed Firefox returns on a OS with `de-DE`:
![image](https://github.com/user-attachments/assets/ffcaf7aa-a0df-47fa-8be8-a36eb42f4fe4)

That's what Camoufox shows when forcing the locale to `de-DE` or letting `geoip` find it:
![image](https://github.com/user-attachments/assets/721c0ed4-946d-487a-be8a-a11ebcc5fa57)

While this makes no problem for a WAF or other bot-detection, the web-application on the other end might freak out.

### What I already tried:
``` Python
        "geoip": False,
        "locale": "de-DE",
        "firefox_user_prefs": {
            "intl.accept_languages": "de,en-US,en",
        }
```

When setting `locale=de-DE, en-US`, the Camoufox is English _only_. When trying to set `locale=de` it requests a region.

**Comment by daijro** [comment]
Hello,

I believe the issue is related to `intl.accept_languages` being overridden with the locale set by the `locale` properties. This was originally done so the locales could match the internationalization API (#16), but I guess this can be handled separately in the Python library now. I'll work on a fix.

**Reply by daijro** [reply]
Has been added in v130.0.1-beta.12!

Documentation has been added here: https://camoufox.com/python/usage
